docs/backend/guides/understanding-milmove-routing.md

```diff
@@ -0,0 +1,115 @@
+# Understanding MilMove Routing
+
+This document is a brief summary on how routing on MilMove works.
+
+## Routing
+
+The routing is initialized in `serve.go`, which calls `InitRouting` inside of `routing_init.go`. This initializes the routing that will be used by the MilMove application. This sets up routing for all 3 sites, the MilMove application, office, and admin. Perhaps this not ideal, but this is how it works as of the writing of this document (December 2022).
```

Comment:
Might be helpful to add, it also sets up routing for Prime API and Support API?

Reply:
Also, I like to sometimes add a simple definition/link to get people started if they really are new to a concept. 

"Routing is a mechanism where HTTP requests are routed to the code that handles them. It is how we determine what should happen when a user visits a certain URL."